src/test/HostActivationTests/FrameworkResolution/RollForwardOnNoCandidateFxMultipleFrameworks.cs

```diff
@@ -134,17 +134,22 @@ public RollForwardOnNoCandidateFxMultipleFrameworks(SharedTestState sharedState)
                         .WithApplyPatches(applyPatches)
                         .Version = versionReference),
                 resolvedFramework,
-                commandResult => commandResult.Should().Fail().And.FailedToSoftRollForward(MicrosoftNETCoreApp, "5.1.3", versionReference));
+                commandResult => commandResult.Should().Fail().And.FailedToSoftRollForward(MicrosoftNETCoreApp, "5.1.1", versionReference));
         }
 
         // Soft roll forward from inner framework reference [specified] to app's 6.1.1-preview.0 (defaults)
         [Theory]
-        [InlineData("6.0.0", null, null, null)]    // Can't roll forward from release to pre-release
+        // 3.0 change:
+        // 2.* - release would never roll forward to pre-release
+        // 3.* - release rolls forward to pre-release if there is no available release match
+        [InlineData("6.0.0", null, null, "6.1.1-preview.1")]
         [InlineData("6.0.1-preview.0", null, null, "6.1.1-preview.1")]
         [InlineData("6.1.1-preview.0", null, null, "6.1.1-preview.1")]
+        [InlineData("6.0.1-preview.0", 0, null, null)]
+        [InlineData("6.1.0-preview.0", 0, false, null)]
         [InlineData("6.1.0-preview.0", 0, null, "6.1.1-preview.1")] // This is effectively a bug, the design was that pre-release should never roll on patches
```

Comment:
```suggestion
         [InlineData("6.1.0-preview.0", 0, null, "6.1.1-preview.1")]
```
If it is really a bug it should be fixed.

Reply:
Well, it's a shipped bug, so now a feature.
I'll remove the comment, because the overall pre-release behavior changes make it a moot point anyway. (We're changing the design in this case)